src/docs/development/platform-integration/add-to-app/performance.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Load sequence, performance and memory
+---
+
+This page serves to inform the reader of the breakdown of the steps involved
+to show a Flutter UI. Knowing this, the user can make better informed decisions
+about when to pre-warm the Flutter engine, which operations are possible
+at which stage and the latency and memory costs of those operations.
+
+## Loading Flutter
+
+Both Android/iOS (the 2 supported platforms for integrating into existing
+apps) and full-Flutter apps and add-to-app patterns have a similar sequence
+of conceptual loading steps when showing Flutter UI.
+
+### Finding the Flutter resources
+
+Flutter's engine runtime and your application's compiled Dart code are both
+bundled as shared libraries on Android and iOS. The first step of loading
+Flutter is to find those resources in your .apk/.ipa/.app etc. (along with
+other Flutter assets such as images, fonts, JIT code if applicable).
+
+This happens when you construct a FlutterEngine for the first time on both
+**[Android]({{site.api}}/javadoc/io/flutter/embedding/engine/FlutterEngine.html)**
+and **[iOS]({{site.api}}/objcdoc/Classes/FlutterEngine.html)** APIs.
+
+### Loading the Flutter library
+
+Once found, the engine's shared libraries are memory loaded once per process.
+
+On **Android**, this happens also when the [`FlutterEngine`]({{site.api}}/javadoc/io/flutter/embedding/engine/FlutterEngine.html)
+is constructed since the JNI connectors need to reference the Flutter C++
+library. On **iOS**, this happens when the [`FlutterEngine`]({{site.api}}/objcdoc/Classes/FlutterEngine.html)
+is first run, such as via [`runWithEntrypoint:`]({{site.api}}/objcdoc/Classes/FlutterEngine.html#/c:objc(cs)FlutterEngine(im)runWithEntrypoint:).
```

Comment:
We don't use "via".  https://developers.google.com/style/word-list#letter-V

such as via => for example, when calling

Reply:
Ah, I keep forgetting about that doc

Reply:
Changed here and elsewhere (in the next commit)